lib/node_modules/@stdlib/stats/base/dmeanpn/test/test.dmeanpn.native.js

```diff
@@ -216,14 +212,13 @@ tape( 'the function supports a negative `stride` parameter', opts, function test
 		2.0
 	]);
 
-	N = floor( x.length / 2 );
-	v = dmeanpn( N, x, -2 );
+	v = dmeanpn( 4, x, -2 );
 
 	t.strictEqual( v, 1.25, 'returns expected value' );
 	t.end();
 });
 
-tape( 'if provided a `stride` parameter equal to `0`, the function returns the first element', opts, function test( t ) {
+tape( 'if provided a `strideX` parameter equal to `0`, the function returns the first element', opts, function test( t ) {
```

Comment:
```suggestion
tape( 'if provided a `stride` parameter equal to `0`, the function returns the first element', opts, function test( t ) {
```
there is no need to change the description of the test case.

Reply:
change it for all the test files

Reply:
Resolved